scene/gui/scroll_bar.cpp

```diff
@@ -497,6 +494,22 @@ Size2 ScrollBar::get_minimum_size() const {
 	return minsize;
 }
 
+void ScrollBar::scroll(double p_amount) {
+	double prev_scroll = get_value();
+	set_value(prev_scroll + p_amount);
+	if (!Math::is_equal_approx(prev_scroll, get_value())) {
```

Comment:
Here you could do `double new_scroll = get_value() + p_amount` and do like I commented in `scroll_to`.

Reply:
Ok sorry, this suggestion was wrong .-.
Adding like this will ignore limits. Setting value is more reliable.

Reply:
Restored it, though kept in my change about having `scroll` call `scroll_to`